CONTRIBUTING.md

```diff
@@ -24,7 +24,7 @@ Attend ONNX Steering Committee meetings - they are open to all in the community.
 
 ## Contributing code
 
-You can submit a pull request (PR) with your code. The [SIG](https://github.com/onnx/onnx/blob/master/community/sigs.md) or [Working Group](https://github.com/onnx/onnx/blob/master/community/working-groups.md) that is responsible for the area of the project your PR touches will review it and merge once any comments are addressed.
+You can submit a pull request (PR) with your code. The [SIG](https://github.com/onnx/onnx/blob/main/community/sigs.md) or [Working Group](https://github.com/onnx/onnx/blob/main/community/working-groups.md) that is responsible for the area of the project your PR touches will review it and merge once any comments are addressed.
```

Comment:
I would recommend to use relative links, which work well IMHO. Path starts in directory of the file. And you then don't need to mention the branch either.

Reply:
Thank you for the suggestion! Yes it can reduce the maintenance cost in the future and make the document cleaner. Actually I was considering the same thing, but I think for README.md we might need to keep it as absolute path -- the absolute path can make the [links](https://pypi.org/project/onnx/) in PyPI workable. We changed it into absolute path since ONNX 1.9 by this PR: https://github.com/onnx/onnx/pull/3386. Except README.md, I went thru all documents and changed the absolute path into relative path.